src/main/java/org/littleshoot/proxy/mitm/CertificateHelper.java

```diff
@@ -80,20 +80,6 @@
      */
     private static final String SIGNATURE_ALGORITHM = is32BitJvm() ? "SHA256" : "SHA512" + "WithRSAEncryption";
 
-    /**
-     * Uses the non-portable system property sun.arch.data.model to help
-     * determine if we are running on a 32-bit JVM. Since the majority of modern
-     * systems are 64 bits, this method "assumes" 64 bits and only returns true
-     * if sun.arch.data.model explicitly indicates a 32-bit JVM.
-     *
-     * @return true if we can determine definitively that this is a 32-bit JVM,
-     *         otherwise false
-     */
-    private static boolean is32BitJvm() {
-        Integer bits = Integer.getInteger("sun.arch.data.model");
-        return bits != null && bits == 32;
-    }
-
```

Comment:
**Note:** This method is taken from @jekh browsermob-proxy/mitm. I'm unsure if it is valid on Android. This has to be verified by measure.

But it's not a problem within this PR fixing SonarQube warnings.